pkg/manager/manager.go

```
@@ -670,8 +770,87 @@
+				if fileExists(localFilePath) {
+					currentHash, err := fileContentSHA256(localFilePath)
+					if err != nil {
+						return fmt.Errorf("failed to compute hash of local file: %w", err)
+					}
+
+					if currentHash != ruleToUpgrade.ContentSHA256 {
+						// File has been modified locally
+						fmt.Printf("Warning: Local file %s has been modified since installation.\n", ruleToUpgrade.LocalFiles[0])
+						fmt.Print("Do you want to proceed and overwrite your local changes? (y/N): ")
+						var answer string
+						fmt.Scanln(&answer)
```

golangci-lint (1.64.8)

Error return value of `fmt.Scanln` is not checked

(errcheck)